policies-procedures-and-docs/README.md

```diff
@@ -0,0 +1,6 @@
+# policies-procedures-and-docs
```

Comment:
We should probably have the Open Seeds procedure within the Open Seeds folder, make them proper page, and add them to the menu

Reply:
**Question** - Should info about honorarium & invoices (i.e `payments` dir) go in this folder too or remain at the root of openseeds?